omero/iviewer/index.html

```diff
@@ -0,0 +1,136 @@
+            <h2>Features</h2>
+            <hr>
+        </div>
+        <hr class="invisible"/>
+        <div class="row small-up-1 medium-up-4">
+            <div class="figure-feature column">
+                <i class="icon-fa-blue fa fa-search-plus fa-2x"></i>
+                <h5>Zoom, Pan and Rotate</h5>
+                <p>
+                    Navigate around images by zooming, panning and
+                    rotating while still keeping track of where
+                    you are in the full image.
+                </p>
+            </div>
+             <div class="figure-feature column">
+                <i class="icon-fa-blue fa fa-arrows-alt fa-2x"></i>
+                <h5>Scroll Z and Time</h5>
+                <p>
+                    Move the image viewport through Z and Time
+                    dimensions.
+                </p>
+            </div>
+            <div class="figure-feature column">
+                <i class="icon-fa-blue fa fa-bars fa-2x"></i>
+                <h5>Live Scalebars</h5>
+                <p>
+                    Pixel size metadata gives live scalebars that
+                    update as you zoom and can be repositioned in the
+                    viewport.
+                </p>
+            </div>
+            <div class="figure-feature column">
+                <i class="icon-fa-blue fa fa-sliders fa-2x"></i>
+                <h5>Adjust rendering settings</h5>
+                <p>
+                    Toggle channels on and off, change colors and adjust
+                    levels.
+                </p>
+            </div>
+            <div class="figure-feature column">
+                <i class="icon-fa-blue fa fa-table fa-2x"></i>
+                <h5>Use Lookup Tables</h5>
+                <p>Render images with lookup tables.</p>
+            </div>
+            <div class="figure-feature column">
+                <i class="icon-fa-blue fa fa-adjust fa-2x"></i>
+                <h5>Invert intensity</h5>
+                <p>Invert the intensity of a channel for display.</p>
+            </div>
+            <div class="figure-feature column">
+                <i class="icon-fa-blue fa fa-signal fa-2x"></i>
+                <h5>View histograms</h5>
+                <p>Full support for histograms of image intensity.</p>
+            </div>
+            <div class="figure-feature column">
+                <i class="icon-fa-blue fa fa-calendar fa-2x"></i>
+                <h5>View metadata</h5>
+                <p>
+                    Both acquisition and import date is available in the
+                    'info' tab.
+                </p>
+            </div>
+            <div class="figure-feature column">
+                <i class="icon-fa-blue fa fa-copy fa-2x"></i>
+                <h5>Open multi-selected images</h5>
+                <p>
+                    Support for multiple side-by-side viewers and the ability
+                    to synchronize settings across datasets.
```

Comment:
I think "datasets" could be confusing. Maybe just "...synchronize settings between them".